src/ForDev.tsx

```
@@ -53,123 +53,138 @@ const BaudRateList = [
     const rmvCS = (rmvSerial: string) => {
-        setConnectedSerialList(prevList => prevList.filter(n => n !== rmvSerial));
-    }
+        setConnectedSerialList((prevList) =>
+            prevList.filter((n) => n !== rmvSerial),
+        );
+    };
 
     const getPorts = async () => {
-        let ports = await invoke("plugin:ardeck|get_ports") as SerialPortInfo[];
+        // let ports = await invoke("plugin:ardeck|get_ports") as SerialPortInfo[];
+        let ports = await invoke.ardeck.getPorts();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'ports' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)